index.html

```diff
@@ -475,7 +485,7 @@ <h2>
         </li>
         <li>
           If <var>signal</var>’s <a>aborted flag</a> is set, then
-          reject <var>p</var> with <code>"<a>AbortError</a>"</code>
+          reject <var>p</var> with an <code>"<a>AbortError</a>"</code>
```

Comment:
Same with the rest... you don't need the code :) 
```suggestion
          reject <var>p</var> with an "<a>AbortError</a>"
```